tests/integ/sagemaker/jumpstart/conftest.py

```diff
@@ -113,6 +133,37 @@ def _teardown():
     bucket = s3_resource.Bucket(test_cache_bucket)
     bucket.objects.filter(Prefix=test_suite_id + "/").delete()
 
+    # delete private hubs
+    _delete_hubs(sagemaker_session)
+
+
+def _delete_hubs(sagemaker_session):
+    # list Hubs created by PySDK integration tests
+    list_hub_response = sagemaker_session.list_hubs(
+        name_contains=os.environ[ENV_VAR_JUMPSTART_SDK_TEST_HUB_NAME]
+    )
+
+    for hub in list_hub_response["HubSummaries"]:
+        if hub["HubName"] != SM_JUMPSTART_PUBLIC_HUB_NAME:
+            # delete all hub contents first
+            _delete_hub_contents(sagemaker_session, hub["HubName"])
+            sagemaker_session.delete_hub(hub["HubName"])
```

Comment:
careful, if there are more than one hub to delete, you might get throttled by the TPS limit of 1.

Reply:
there'll be only 1 hub at a time